<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QureyDepartment</class>
 <widget class="QDialog" name="QureyDepartment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>部门查询</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>521</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="choice">
        <item>
         <property name="text">
          <string>请选择查询方式</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>部门编号</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>部门名称</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>部门职能</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>负责人编号</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputInformation">
        <property name="placeholderText">
         <string>请输入查询信息</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="searchButton">
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tableViewInformation"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
